docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -222,62 +224,113 @@ endif::[]
-* You're logged in to the {product-api-designer} web console at {service-url-api-designer}[^].
+* You're logged in to the {product-long-api-designer} web console at {service-url-api-designer}[^].
 * You've created or imported an {product-api-designer} design.
 * You can access a running {product-registry} instance in your organization.
+For more information, see {base-url}{getting-started-url-registry}[Getting started with {product-long-registry}^].
 
 .Procedure
-. In the *{product-api-designer}* designs page of the web console, select the {product-api-designer} design that you want to export.
-. Click *Edit* to open the design editor.
-. From the *Actions* menu, click *Export to {registry}*.
-. Complete the form to specify where the new design should be saved.
+. In the *API and Schema Designs* page of the {service-url-api-designer}[{product-long-api-designer} web console^], select the design that you want to export.
+. Click the options icon (three vertical dots), and click *Export to {registry}*.
+. Complete the form to specify where to save the new design.
 +
-NOTE: If the design was originally imported from {product-registry}, the fields are prepopulated with the details of the original {product-registry} instance, and the *Version* is incremented.
+NOTE: If the design was originally imported from {product-long-registry}, the fields are prepopulated with the details of the original {product-registry} instance, and the *Version* is incremented.
 +
 .. *Registry instance*: Select the required instance from the list.
-.. *Group*: Enter an optional unique group name such as `my-org` to organize the artifact in a named collection. Each group contains a logically related set of schemas or API designs, typically managed by a single entity, belonging to a particular application or organization.
+.. *Group*: _Optional:_ Enter a unique group name such as `my-org` to organize the artifact in a named collection.
+Each group contains a logically related set of schemas or API designs, typically managed by a single entity, belonging to a particular application or organization.
 +
 NOTE:  Specifying a group is optional when using the web console: {registry} generates a `default` group automatically.
 +
-.. *Artifact ID*: Enter an optional unique ID for this artifact, such as `my-ID`. If you don't specify a unique artifact ID, {registry} generates one automatically as a UUID.
-.. *Version*: Specify the version number.
+.. *Artifact ID*: _Optional:_ Enter a unique ID for this artifact, such as `my-ID`.
+If you don't specify a unique artifact ID, {registry} generates one automatically as a UUID.
+.. *Version*:  _Optional:_ Specify the version number.
+If you don't specify a version number, {registry} sets the version to 1.
 +
 [.screencapture]
 .Exporting an API design
 image::export-api-designer-design.png[Image of page to export an API design]
 +
 . Click *Export*.
+The exported design is listed on the artifacts page of the specified {registry} instance.
```

Comment:
Eliminate passive voice, if possible.

Reply:
System activity is acceptable use of passive voice.